CHANGELOG.md

```diff
@@ -48,7 +48,8 @@
 * [ENHANCEMENT] Store-gateway: use more efficient chunks fetching and caching. #4255
 * [ENHANCEMENT] Query-frontend and ruler: add experimental, more performant protobuf internal query result response format enabled with `-ruler.query-frontend.query-result-response-format=protobuf`. #4331
 * [ENHANCEMENT] Ruler: increased tolerance for missed iterations on alerts, reducing the chances of flapping firing alerts during ruler restarts. #4432
-* [ENHANCEMENT] Querier and store-gateway: optimized `.*` and `.+` regular expression label matchers. #4432
+* [ENHANCEMENT] Optimized `.*` and `.+` regular expression label matchers. #4432
```

Comment:
Note to reviewers: I modified this because it's not true it applies to querier, but applies to ingesters and store-gateways and query-frontend (query sharding). So, to simplify it, I've just removed to which components apply.